openfisca_core/scripts/run_test.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+
+import argparse
+import logging
+import sys
+import os
+import importlib
+
+from openfisca_core.tools.test_runner import run_tests
+from openfisca_core.tools import detect_country_packages
+
+
+def build_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('path', help = "paths (files or directories) of tests to execute", nargs = '+')
+    parser.add_argument('-c', '--country_package', action = 'store', help = 'country package to use to run the test. If not provided, an automatic detection will be atempted.')
```

Comment:
`attempted`.

A basic idea of what the automatic detection will look for would be welcome.